strings.py

```diff
@@ -1,9 +1,15 @@
 cannot_change_category = 'Category cannot be changed'
 cannot_change_state = 'State cannot be changed'
+cannot_change_type = 'Type cannot be changed'
 category_not_found = 'Category does not exist'
 criterion_not_found = 'Criterion does not exist'
 invalid_state = 'State code is invalid'
 link_not_found = 'Link does not exist'
+require_type = 'type is required to create a new link. type must be one of ("resource_link", "honorable_mention", or "innovative_policy_idea")'  # noqa: E501
```

Comment:
This is a nitpick -- would you mind changing the variable name of this string to be something like `require_link_type`, since the text is specific to links? That way if we have another data model that also has types, we know not to use this string.